noir_stdlib/src/array.nr

```diff
@@ -1,16 +1,5 @@
 #[builtin(array_len)]
 fn len<T>(_input : [T]) -> comptime Field {}
 
-// insertion sort - n.b. it is a quadratic sort
-fn sort<T>(mut a: [T]) -> [T] { 
-    for i in 1..len(a) {
-        for j in 0..i {
-            if(a[i] < a[j]) {
-                let c = a[j];
-                a[j] = a[i];
-                a[i]= c;
-            }
-        };
-    };
-    a
-}
+#[builtin(arraysort)]
+ fn sort<T>(_a: [T]) -> [T] {}
```

Comment:
We should also add the explicit `N` here since #562 is still open:
```rs
fn sort<T, N>(_a: [T; N]) -> [T; N] {}
```

Reply:
Done